packages/components/src/template-registry.ts

```diff
@@ -29,4 +30,10 @@ export default interface HdsComponentsRegistry {
   // Helpers
   'hds-link-to-models': typeof HdsLinkToModelsHelper;
   'hds-link-to-query': typeof HdsLinkToQueryHelper;
+  HdsBadgeComponent: typeof HdsBadgeIndexComponent;
+
+  // Badge
+  'Hds::Badge': typeof HdsBadgeIndexComponent;
+  'hds/badge': typeof HdsBadgeIndexComponent;
+  'HdsBadge': typeof HdsBadgeIndexComponent;
 }
```

Comment:
+1 to the above comments, with the addition that the last entry should be `HdsBadge`. they should be placed alphabetically, but component and before the helpers, indeed

```suggestion
  // Badge
  'Hds::Badge': typeof HdsBadgeIndexComponent;
  'hds/badge': typeof HdsBadgeIndexComponent;
  HdsBadge: typeof HdsBadgeIndexComponent;

    // Helpers
  'hds-link-to-models': typeof HdsLinkToModelsHelper;
  'hds-link-to-query': typeof HdsLinkToQueryHelper;
}
```